CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,7 @@
 ### Bug fixes
 
 - [Continuous Replication] Fix VHD size guess for empty files [#4105](https://github.com/vatesfr/xen-orchestra/issues/4105)  (PR [#4107](https://github.com/vatesfr/xen-orchestra/pull/4107))
+- [Menu] XOA: Display icon when no notifications nor updates [#4012](https://github.com/vatesfr/xen-orchestra/issues/4012) (PR [#4068](https://github.com/vatesfr/xen-orchestra/pull/4068)
```

Comment:
I'm not sure I'd understand it as a user. Maybe "Fixed empty slot when menu is collapsed".